test/e2e/servicebindingrequest_test.go

```diff
@@ -59,25 +59,14 @@ func TestAddSchemesToFramework(t *testing.T) {
 	require.Nil(t, framework.AddToFrameworkScheme(pgsqlapis.AddToScheme, &dbList))
 
 	t.Run("end-to-end", func(t *testing.T) {
+		// scenario-1
 		t.Run("scenario-db-app-sbr", func(t *testing.T) {
 			ServiceBindingRequest(t, []Step{DBStep, AppStep, SBRStep})
 		})
-		t.Run("scenario-app-db-sbr", func(t *testing.T) {
```

Comment:
Let's implement just the 2 existing scenarios here and add the other permutations when this is merged.